internal/persistence/opensearch/models.go

```diff
@@ -255,6 +255,18 @@ const (
 	OpenSearchMajorVersionV1 OpenSearchMajorVersion = "V1"
 )
 
+func (e OpenSearchMajorVersion) IsDowngradeTo(other OpenSearchMajorVersion) bool {
+	if !e.IsValid() || !other.IsValid() {
+		return false
+	}
+
+	if e == OpenSearchMajorVersionV1 && other == OpenSearchMajorVersionV2 {
+		return true
+	}
+
+	return false
```

Comment:
Could perhaps flip this so that we default the fallback to true, in case someone adds a V3 in the future without modifying this function

Alternatively (more hacky): strip the "V" prefix and compare integers

Reply:
Gikk for hacken, mindre å huske på når v3 kommer